index.html

```diff
@@ -17,16 +17,21 @@
     <header>
         <nav class="container-nav">
             <a href="#" class="logo-name">Federica Ulzurrun</a>
-            <div class="navlog">
-                <img src="img/navhamburger.png" alt="20px">
-            </div>
-            <ul class="nav-desk">
-                <li class="page-parts">portfolio</li>
-                <li class="page-parts">about</li>
-                <li class="page-parts">contact</li>
+            <ul class="nav-menu">
+                <li class="page-parts"><a class="nav-link" href="#portfolio">portfolio</a></li>
+                <li class="page-parts"><a href="#about" class="nav-link">about</a></li>
+                <li class="page-parts"><a href="#cont" class="nav-link">contact</a></li>
             </ul>
+            <div class="nav-log">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
```

Comment:
- [ ] It appears that you forgot to disable this on bigger screens with enough `viewport` to array the navigation links. Kindly remove the menu icon from screen sizes with a `min-width` of `768px` for a more quality user experience and smooth accessibility. 

![image](https://user-images.githubusercontent.com/99715640/223247646-4c883eaa-c31a-4bea-9dfa-b300e4c74fc9.png)